Connect/AzureStack.Connect.psm1

```diff
@@ -93,7 +97,7 @@ function Add-AzureStackAzureRmEnvironment
 
     $azureEnvironmentParams = @{
         Name                                     = $Name
-        ActiveDirectoryEndpoint                  = $endpoints.authentication.loginEndpoint.TrimEnd('/') + "/" + $AadTenant + "/"
+        ActiveDirectoryEndpoint                  = $endpoints.authentication.loginEndpoint.TrimEnd('/') + "/"
         ActiveDirectoryServiceEndpointResourceId = $endpoints.authentication.audiences[0]
         AdTenant                                 = $AadTenant
```

Comment:
Can we make sure that  $AadTenant is in GUID format? This used to work with the tenant name like domain.onmicrosoft.com, not any more